charts/velero/templates/upgrade-crds.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if .Release.IsUpgrade }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "velero.fullname" . }}-upgrade-crds
```

Comment:
Will this create a fixed job name with no unique components? My concern is that there is no TTL specified on this job which means that it will persist after completion. If the user attempts another upgrade, this will fail as the job name already exists.

Reply:
Ref to https://helm.sh/docs/topics/charts_hooks/#hook-deletion-policies
By default, the `helm.sh/hook-delete-policy` is `before-hook-creation`, which means the previous upgrade Job would be deleted before this new Job is launched.

I could verify it again.

Reply:
Ahh, sorry, I wasn't aware of that. If helm already manages this then that's fine :) thanks for the explanation!